eva-server/src/main/java/uk/ac/ebi/eva/server/Utils.java

```diff
@@ -43,13 +48,28 @@
     public static PageRequest getPageRequest(QueryOptions queryOptions) {
         int limit = (int) queryOptions.get("limit");
         int skip = (int) queryOptions.get("skip");
```

Comment:
Set default values when the properties are undefined. Otherwise these casts will throw an exception. For consistency, they can be the same as in https://github.com/EBIvariation/eva-ws/pull/37/files#diff-45d2055383ecc49840bd0966d60042f9R62